### DISCUSSION TITLE
Does ORFS have a power optimization feature?

### DISCUSSION THREAD

**Discussion by dalyles** [The-OpenROAD-Project/OpenROAD-flow-scripts#1694] [Q&A]
I wanted to ask if OpenROAD Flow Scripts has any power optimization features.

Let's say that you're trying to generate a SoC using ORFS and you want to minimize
power dissipation, is there any feature in the scripts or in the GUI that allows you to
optimize (or minimze) power consumption?

Thanks!

**Comment by maliberty** [comment]
Its not been a major focus but you can try the RECOVER_POWER option for ORFS.

**Comment by dalyles** [comment]
Ok, I will look into this,

thank you!
